src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -5,6 +5,8 @@ module Webui::NotificationHelper
   MAXIMUM_DISPLAYED_AVATARS = 6
 
   def truncate_to_first_new_line(text)
+    return '' if text.blank?
```

Comment:
I smell a missing spec...

Reply:
Exactly, the specs didn't cover those cases. I added them now